pkg/service/backup/service_backup_integration_test.go

```diff
@@ -2315,7 +2311,7 @@ func TestBackupListIntegration(t *testing.T) {
 func TestBackupAlternatorIntegration(t *testing.T) {
 	const (
 		testBucket     = "backuptest-alternator"
-		testTable      = "table-with-dash"
+		testTable      = "table-with.dash-and.dot.-"
```

Comment:
@Michal-Leszczynski can you add the underscore, uppercase chars and digits to the testTable name ? 
It would fulfill this https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html#HowItWorks.NamingRules then.

Reply:
Done, waiting for tests to pass.